### ISSUE TITLE
What is license of content packs?

### ISSUE THREAD

**Issue by Hubbitus** [RigsOfRods/rigs-of-rods#542]
Hello.

On page http://www.rigsofrods.com/wiki/pages/Starting_RoR_under_Linux present instruction how to download 2 possible packs for game.

I package RoR for Fedora and need specify license for that. Could you please say what it is or where I could find such information?

**Comment by Hiradur** [comment]
Unfortunately most of these mods don't state a license and thus have to be considered non-free. Most mod makers use very restrictive licenses, right of the bat I don't know of any libre mod.

**Comment by Hubbitus** [comment]
Hm. It is bad news. Where I could find information about that per module?

> Unfortunately most of these mods don't state a license

Couldn't it be then otherwise treated as public domain? How I could ask their authors?

**Comment by Hubbitus** [comment]
Do you force or ask authors pick license before publication?

**Comment by Hiradur** [comment]
> Couldn't it be then otherwise treated as public domain? How I could ask their authors?

No, everything without license information is automatically "all rights reserved by author(s)".


> Do you force or ask authors pick license before publication?

No, we don't and we don't have control over the website rigsofrods.com so there is little room for such things atm.

**Comment by Hubbitus** [comment]
Ogm. Who is control rigsofrods.com? How I can contact them with that question?

**Comment by Hiradur** [comment]
tdev is in control of rigsofrods.com, contact info here: https://github.com/RigsOfRods/rigs-of-rods/blob/master/AUTHORS.md

However, I think you would be wasting your time. The open source spirit has never been big in the community. Many authors of the mods are inactive and if there is no license information publicly available I don't think tdev could do something about it.

**Comment by DirtGamer301** [comment]
This might be slightly off-topic but here's a general license question and I think I know the answer but still wanted to make sure: There's no way to allow editing and releasing of edits with credits under the same license, but only for Rigs of Rods, right?

**Comment by Hiradur** [comment]
> There's no way to allow editing and releasing of edits with credits under the same license, but only for Rigs of Rods, right?

No, unless a term like this would be included in the requirements for uploads.

**Comment by DirtGamer301** [comment]
So if I'd say "license xy mods only allowed to be upload on rigsofrods.com" it'd be legally correct?

**Comment by Hubbitus** [comment]
Yes, much more convenient to users sign some agreement what their uploaded work will be licensed in some way... It is in ideal way.
But now at least just field for each work under what license it distributed will be highly appreciated.
Then I and other package maintainers will be able choose such items which may be included in distribution. By fact many (most?) authors which upload their work for download without charges will not argue if it will be more widely distributed.

**Comment by Hiradur** [comment]
> So if I'd say "license xy mods only allowed to be upload on rigsofrods.com" it'd be legally correct?

Yes, but another problem is that mod makers may not like permissive licenses, many mods are released as "modifications of this mod only with permission by author" which is rather restrictive.

**Comment by Hubbitus** [comment]
Much more problem what I do not see where I can read such meta information! If it will be in some database, we then be able choose and filter only free.

**Comment by monwarez** [comment]
I think that could be great to have at least one car and one real map with FOSS license
